backend/grants/admin.py

```diff
@@ -316,9 +319,12 @@ def user_display_name(self, obj):
         return obj.email
 
     def save_form(self, request, form, change):
-        if (
-            form.cleaned_data["status"] == Grant.Status.approved
-            and form.cleaned_data["status"] != form.initial["status"]
+        # If the status, country_type or approved_type changes and the grant is approved
+        # we need to recalculate the totals
+        if form.cleaned_data["status"] == Grant.Status.approved and (
+            form.cleaned_data["status"] != form.initial["status"]
+            or form.cleaned_data["country_type"] != form.initial["country_type"]
+            or form.cleaned_data["approved_type"] != form.initial["approved_type"]
         ):
             conference = form.cleaned_data["conference"]
             form.instance.ticket_amount = conference.grants_default_ticket_amount
```

Comment:
grants_default_ticket_amount? I don't 100% understand this ticket_amount? why we need it?

Reply:
it's the cost of the person, it's lower than the selling price... It's the amount that we pay for the person to come